pages/docs/couple-your-code/couple-your-code-implicit-coupling.md

```diff
@@ -49,7 +49,8 @@ double dt; // actual time step size
 ```
 
 ```cpp
-preciceDt = precice.initialize();
+precice.initialize();
+preciceDt = precice.getMaxTimeStepSize();
```

Comment:
Move before the "min" below.

Reply:
Done. 1a983eb5511ad3ccf867472d30b537b9aee9ddff